compute_endpoint/globus_compute_endpoint/endpoint/config/utils.py

```diff
@@ -63,7 +63,7 @@ def _load_config_py(conf_path: pathlib.Path) -> Config | None:
         raise
 
 
-def _read_config_yaml(config_path: pathlib.Path) -> str:
+def _read_config_file(config_path: pathlib.Path) -> str:
```

Comment:
Learning from our colleague BenC's strictness about PR change isolation (in Parsl-land), this name refactor could be its own PR.  Not a blocker, just observing.

Reply:
I modified the function name because, as of this PR, the function is used to load both `.yaml` and `.py` config files.

Reply:
*nod; I get it.  But it's also generically doing that already and could (rightfully) be its own PR prior or after this one.  Either way, not a big deal, just observing it in context of Parsl's discipline on this matter.